src/hub.js

```diff
@@ -9,7 +9,7 @@ patchWebGLRenderingContext();
 
 import "aframe-xr";
 import "./vendor/GLTFLoader";
-import "networked-aframe";
+import "networked-aframe/src/index";
```

Comment:
whats the deal with this change?

Reply:
networked-aframe's package.json exposes the bundled version of the library. This makes debugging harder because no sourcemaps are generated. This is a temporary fix until we change networked-aframe's package.json

Reply:
I've created a PR: https://github.com/networked-aframe/networked-aframe/pull/127